src/content/en/updates/posts/2016/03/access-usb-devices-on-the-web.markdown

```diff
@@ -0,0 +1,370 @@
+---
+layout: updates/post
+title: "Access USB devices on the Web"
+description: "The WebUSB API makes USB safer and easier to use by bringing it to the Web."
+featured_image: /web/updates/images/2016-03-02-access-usb-devices-on-the-web/web-usb-hero.jpg
+published_on: 2016-08-12
+updated_on: 2016-08-12
+authors:
+  - beaufortfrancois
+tags:
+  - news
+  - USB
+  - WebUSB
+  - IoT
+  - Arduino
+  - origintrials
+---
+
+If I said plain and simple "USB", there is a good chance that you will
+immediately think of keyboards, mice, audio, video and storage devices. You're
+right but you'll find other kinds of Universal Serial Bus (USB) devices out
+there.
+
+These non-standardized USB devices require hardware vendors to write native
+drivers and SDKs in order for you (the developer) to take advantage of them.
+Sadly this native code has historically prevented these devices from being used
+by the Web. And that's one of the reasons the WebUSB API has been created: to
+provide a way to expose USB device services to the Web. With this API, hardware
+manufacturers will be able to build cross-platform JavaScript SDKs for their
+devices.
+But most importantly this will **make USB safer and easier to use by bringing
+it to the Web**.
+
+Let's see what you could expect with the WebUSB API:
+
+1. Buy a USB device,
+2. Plug it into your computer,
+3. A notification appears right away, with the right website to go to for this device.
+4. Simply click on it. Website is there and ready to use!
+5. Click to connect and a USB device chooser shows up in Chrome, where you can pick your device.
+6. Tada!
+
+What could it have been after you've plugged your USB device?
+
+- Read a box, label, or search online for website -- possibly ending up on the wrong website.
+- Have to install a native application.
+- Is it supported on my operating system? Make sure you download the "right" thing.
+- Scary OS prompts popup and warn you about installing drivers/applications from the Internet.
+- Malfunctioning code harms the whole computer, the Web is built to [contain malfunctioning websites](https://www.youtube.com/watch?v=29e0CtgXZSI).
+- Only use the USB device once? On the Web, the website is gone once you closed tab. On a computer the code sticks around.
+
```

Comment:
General comment about lists and procedures. Best practice is considered to be making sure they follow a parallel grammatical structure. When a bullet or step has multiple sentences it's the first sentence that follows the parallelism. If an item can't be easily rewritten, it may be because it's a continuation of the previous item. If so, consider merging.